internal/coreinternal/textutils/encoding.go

```diff
@@ -14,14 +14,16 @@ import (
 )
 
 var encodingOverrides = map[string]encoding.Encoding{
-	"utf-16":   unicode.UTF16(unicode.LittleEndian, unicode.IgnoreBOM),
-	"utf16":    unicode.UTF16(unicode.LittleEndian, unicode.IgnoreBOM),
-	"utf-8":    unicode.UTF8,
-	"utf8":     unicode.UTF8,
-	"ascii":    unicode.UTF8,
-	"us-ascii": unicode.UTF8,
-	"nop":      encoding.Nop,
-	"":         unicode.UTF8,
+	"utf-16":    unicode.UTF16(unicode.LittleEndian, unicode.IgnoreBOM),
+	"utf16":     unicode.UTF16(unicode.LittleEndian, unicode.IgnoreBOM),
+	"utf-8":     unicode.UTF8,
+	"utf8":      unicode.UTF8,
+	"utf-8-raw": UTF8Raw,
+	"utf8-raw":  UTF8Raw,
+	"ascii":     unicode.UTF8,
+	"us-ascii":  unicode.UTF8,
+	"nop":       encoding.Nop,
+	"":          UTF8Raw,
```

Comment:
Why change the default here from "utf-8" to "utf-8-raw"?

Note that this doesn't change the default encoding used by File Log receiver, as that is configured [elsewhere](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/aa00ed11ae6a39447e069200c7f39ae88f32f35b/pkg/stanza/fileconsumer/config.go#L37).

Reply:
My mistake, I think I should have only changed the default encoding for the `filelog` receiver. Thank you for pointing it out.

Reply:
> If we want to change the default encoding, we should do this in stages, using a [feature gate](https://github.com/open-telemetry/opentelemetry-collector/blob/e9f3dec3c696a429bf860247f530dc58476b7d64/featuregate/README.md).

Indeed. However I am open to changing the default value or leaving it as is. :thinking:

Reply:
I lean towards not changing the default. Seems fine to provide the alternative but I don't see why it should immediately become a new default.